packages/dx-scheduler-core/src/types/horizontal-rect.types.ts

```diff
@@ -0,0 +1,40 @@
+import { PureComputed } from '@devexpress/dx-core';
+import { Multiline, CellElement, AppointmentDate } from './scheduler-core.types';
+import { ViewCellData, TakePrevious } from './all-day-panel.types';
+import { EndDate } from './appointment-form.types';
+
+export interface HorizontalPayload {
+  multiline: Multiline;
+  viewCellsData: ViewCellData[][];
+  cellElements: CellElement[][];
+}
+
+export interface Coordinates {
```

Comment:
One more common question: sometimes we use the 'I' prefix in an interface name. `export interface IDependency {}`, for instance. Is there any reason why?

Reply:
IDependency is to be renamed, we should not use the 'I' prefix for consistency with current d.ts declaration

Reply:
OK. We have some 'I...' interfaces in demos. `with-components` hoc contains the same one.

Reply:
Yes, I'll rename it after all ts PRs are merged